typedb-src/modules/ROOT/pages/quickstart-guide.adoc

```diff
@@ -0,0 +1,534 @@
++
+[,typeql]
+----
+match
+    $p isa person,
+        has full-name $fn;
+    $fn contains "Kevin";
+insert
+    $p has email "kevin@gmail.com";
+----
+. Run the query.
+. Commit the transaction.
+--
+====
+
+Now Kevin has more than one email.
+
+[#_update_query__change_the_newly_added_email_for_kevin]
+=== Update the newly added email for Kevin
+
+The following query updates the new email "kevin@gmail.com" for every person with a full name that contains "Kevin"
+to be "kevin2@gmail.com" instead.
+We have only one such person in the IAM database we have created above.
+
+[,typeql]
+----
+match
+    $p isa person,
+        has full-name $fn,
+        has email $e;
+    $fn contains "Kevin";
+    $e = "kevin@gmail.com";
+delete $p has $e;
+insert $p has email "kevin2@gmail.com";
+----
+
+To try it, choose a TypeDB Client below and follow the steps.
+
+[tabs]
+====
+TypeDB Console::
++
+--
+Use the following command in a local terminal:
+[,bash]
+----
+typedb console --command="transaction try-iam data write" --command='match $p isa person, has full-name $fn, has email $e; $fn contains "Kevin"; $e = "kevin@gmail.com"; delete $p has $e; insert $p has email "kevin2@gmail.com";' --command="commit"
+----
+--
+TypeDB Studio::
++
+--
+. Select `data` session and `write` transaction in the top toolbar.
+. Open a new tab and insert the following TypeQL query:
++
+[,typeql]
+----
+match
+    $p isa person,
+        has full-name $fn,
+        has email $e;
+    $fn contains "Kevin";
+    $e = "kevin@gmail.com";
+delete $p has $e;
+insert $p has email "kevin2@gmail.com";
+----
+. Run the query.
+. Commit the transaction.
+--
+====
+
+[#_delete_query__delete_ownership_of_the_newly_updated_email_from_kevin]
+=== Delete ownership of the newly updated email from Kevin
+
+The following query deletes the ownership over the email "kevin2@gmail.com" for every person with a full name
+that contains "Kevin".
+We have only one such person in the IAM database we have created above.
+
+[,typeql]
+----
+match
+    $p isa person,
+        has full-name $fn,
+        has email $e;
+    $fn contains "Kevin";
+    $e = "kevin2@gmail.com";
+delete $p has $e;
+----
+
+To try it, choose a TypeDB Client below and follow the steps.
+
+[tabs]
+====
+TypeDB Console::
++
+--
+Use the following command in a local terminal:
+[,bash]
+----
+typedb console --command="transaction try-iam data write" --command='match $p isa person, has full-name $fn, has email $e; $fn contains "Kevin"; $e = "kevin2@gmail.com"; delete $p has $e;' --command="commit"
+----
+--
+TypeDB Studio::
++
+--
+. Select `data` session and `write` transaction in the top toolbar.
+. Open a new tab and insert the following TypeQL query:
++
+[,typeql]
+----
+match
+    $p isa person,
+        has full-name $fn,
+        has email $e;
+    $fn contains "Kevin";
+    $e = "kevin2@gmail.com";
+delete $p has $e;
+----
+. Run the query.
+. Commit the transaction.
+--
+====
+
```

Comment:
Condense all of the above examples to just: (1) description and (2) TypeQL code. No references to Console or Studio are necessary.